app/views/layouts/_marketplace_head.haml

```diff
@@ -99,3 +99,7 @@
 
 - unless on_admin?
   = @current_community.custom_head_script.try(:html_safe)
+
+- if on_admin? && IntercomHelper.admin_intercom_respond_enabled?
+  - user_uuid = @current_user.uuid_object.to_s
+  = render partial: "layouts/intercom", locals: { app_id: IntercomHelper.admin_intercom_app_id, email: @current_user.primary_email&.address, user_id: @current_user.id, user_uuid: user_uuid, name: PersonViewUtils.person_display_name(@current_user, @current_community), marketplace_id: @current_community.id, marketplace_uuid: @current_community.uuid_object.to_s, marketplace_url: @current_community.full_url, user_hash: IntercomHelper.user_hash(user_uuid) }
```

Comment:
Why are the locals passed from here? Not saying that this is wrong, but I want to just understand this. 

It would make perfect sense if we would pass these from a controller, but now it just seems that the responsiblity to extract locals is transferred to another view.